### ISSUE TITLE
Two time Scale update is not used in this train script

### ISSUE THREAD

**Issue by mostafaelaraby** [fancoo/WaveGAN-pytorch#6]
The two time scale update rule as mentioned in https://arxiv.org/pdf/1706.08500.pdf is about giving higher learning rate to the discriminator and instead of training the discriminator for 5 steps then updating the generator they are both trained at the same number of iters but with different learning rate ,but in your code you added a comment stating that you used this technique however both optimizers are having the same learning rate and the generator is updated once every 5 steps of discriminator

https://github.com/mazzzystar/WaveGAN-pytorch/blob/3c99f1db34052c39432be9564727dbcfd909df5c/train.py#L50
